packages/peregrine/lib/talons/CreateAccount/useCreateAccount.js

```diff
@@ -74,7 +74,8 @@ export const useCreateAccount = props => {
                         email: formValues.customer.email,
                         firstname: formValues.customer.firstname,
                         lastname: formValues.customer.lastname,
-                        password: formValues.password
+                        password: formValues.password,
+                        is_subscribed: typeof formValues.subscribe === 'boolean' ? formValues.subscribe : false
```

Comment:
Can we just use `!!formValues.subscribe`?

Reply:
I think we can everything. Will fix soon. 

Thank you, @sirugh

Reply:
**done**

@sirugh , could you please approve?

Thank you!